### ISSUE TITLE
erro 'TypeError: range must be set' while previewing document

### ISSUE THREAD

**Issue by joaompinto** [asciidoctor/asciidoctor-vscode#370]
## Description

While trying to preview an Asciidoc document on VSCode the following error is reported:

TypeError: range must be set

## System Information

Version: 1.50.1 (user setup)
Commit: d2e414d9e4239a252d1ab117bd7067f125afd80a
Date: 2020-10-13T15:06:15.712Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.18363

## To Reproduce

Steps to reproduce the issue:
Ctrl + N to create a new file,
paste:
```
= Guide
João Pinto <lamego.pinto@gmail.com>
:source-highlighter: prettify
```
CTRL-K ; M (Select Asciidoc)
CTRL-K ; V

**Comment by danyill** [comment]
Is this a duplicate of #364, closed in #367?

**Comment by joaompinto** [comment]
@danhyun 🤦 yes, will publish the release asap.
